test/kubemark/start-kubemark.sh

```diff
@@ -367,13 +93,15 @@ current-context: kubemark-context"
     --from-file=kernel.monitor="${RESOURCE_DIRECTORY}/kernel-monitor.json"
 
   # Create secret for passing kubeconfigs to kubelet, kubeproxy and npd.
+  # It's bad that all component shares the same kubeconfig.
+  # TODO(mborsz): Migrate all components to separate credentials.
```

Comment:
Can you file an issue for it so that we could better track it?

Reply:
https://github.com/kubernetes/kubernetes/issues/79883